photon-lib/py/photonlibpy/simulation/photonCameraSim.py

```diff
@@ -432,17 +432,18 @@ def distance(target: VisionTargetSim):
         )
 
     def submitProcessedFrame(
-        self, result: PhotonPipelineResult, receiveTimestamp: float | None
+        self,
+        result: PhotonPipelineResult,
+        receiveTimestamp: float = wpilib.Timer.getFPGATimestamp() * 1e6,
```

Comment:
When does this default get evaluated?

Reply:
Good call. I dont think there is a solution to this without drastically changing the typings

Reply:
Could you check if none and then default to getFpgaTimestamp?

Reply:
Thats basically what it is right now without screwing with the type signature of the function to make it do funny things. The intent of this was to remove optional typing from the signature and if that cant be sorted I don't think this is worth worrying about